docs/_docs/code-snippets/k8s/stateful/node-configuration.xml

```diff
@@ -16,11 +16,17 @@
   limitations under the License.
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-    xsi:schemaLocation="http://www.springframework.org/schema/beans         
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
     http://www.springframework.org/schema/beans/spring-beans.xsd">
 
     <!-- tag::config-block[] -->
+    <bean class="org.apache.ignite.kubernetes.configuration.KubernetesConnectionConfiguration" name="k8sCfg">
```

Comment:
I would put this bean definition into the TcpDiscoveryKubernetesIpFinder bean right away. Simplifies readability. Plus, the KubernetesConnectionConfiguration bean is used only by the TcpDiscoveryKubernetesIpFinder bean; thus it should be fine to put one inside of another right away.

Please consider such a change for similar places below.

Reply:
fixed